ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-presubmits.yaml

```diff
@@ -631,6 +631,58 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )4.8-vsphere-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Think we should set this to "false" so that the images are not built. We're building only 4.8 in pre-submits.